### ISSUE TITLE
Search app with no facets/side controls should allow results to be full screen

### ISSUE THREAD

**Issue by EvanBoyle** [EvanBoyle/AzSearch.js#40]
Shouldn't show that facet bar if no facets are defined.

**Comment by m-gagne** [comment]
Was starting to look at this, wouldn't this just be as easy as removing all facet/filter calls and changing the layout (removing the `facetPanel` div & the  `col-md-offset-3` class from the `results_section` container? See this gist for test layout: https://gist.github.com/m-gagne/039d29a55b06b87523b90808ba27c126

**Comment by EvanBoyle** [comment]
Yeah I was just collecting some feedback from someone who was experimenting with the app. I think the real solution to this problem may be to modify the HTML template rather than having the code adapt to this. Given the fact that the user is responsible for laying out the components the way they want, and the sample is just one particular way to achieve that.

**Comment by m-gagne** [comment]
Agreed, in my testing I created a local testing folder and created a copy of the local realestate file. We could start a "tests" or "sample layouts" folder to highlight a few things like this (also handy for testing?)

**Comment by EvanBoyle** [comment]
Yeah some sample layouts would be great. Along the same lines, it would be great to have an example that showed a custom CSS theme. If we're going to maintain multiple samples, it would be nice if we could automate the version update process. Is there some way that we can add a build step or command that updates the azsearch cdn version numbers automatically?

**Comment by m-gagne** [comment]
hrm... first reaction is some sort of bash script + regex, but let me think about this for longer than 15 seconds =)

**Comment by m-gagne** [comment]
What are your thoughts about moving the following under `./samples/realestate`
* realestate.html
* realestate.local.html
* realestate.local-no-filters.html
* AzSearch.css

**Comment by m-gagne** [comment]
It would appear that using the `@latest` syntax when referencing items from the CDN will do the trick!

* `<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/azsearch.js@latest/dist/AzSearch.css">`
* `<script src="https://cdn.jsdelivr.net/npm/azsearch.js@latest/dist/AzSearch.bundle.js"></script>`

**Comment by EvanBoyle** [comment]
ok this is pretty cool. Not sure if we need to worry about the people getting broken due to not using a fixed version though. For example if someone forks and modifies the real estate sample and an API changes while we're still in active dev phase. Probably OK as long as the app generator continues to use a fixed api verison.

**Comment by EvanBoyle** [comment]
I also like the solution of making the modified templates local only. That lets us ignore the versioning problem.

**Comment by m-gagne** [comment]
Yeah I think that would be ok, could also include a note re: `./samples` that they reference the latest, however in production/forks they should reference a specific version.

**Comment by m-gagne** [comment]
So ideally a 'master` type template such as realestate.html & then a local verison + customized verisons (which are always local?)
